engine/crates/federated-graph/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
-name = "grafbase-federated-graph"
+name = "graphql-federated-graph"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0"
```

Comment:
Not MPL2?

Reply:
I don't know, I aligned with the CLI license but I don't have an opinion on that.

Reply:
(if this is implementing a spec as you said before maybe that's not relevant)